torch/_torch_docs.py

```diff
@@ -2936,9 +2936,9 @@ def parse_kwargs(desc):
 Computes the multivariate log-gamma function with dimension :math:`p` element-wise, given by
 
 .. math::
-    \log(\gamma_{p}(a)) = C + \displaystyle \sum_{i=1}^{p} \log\left(\gamma\left(a - \frac{i - 1}{2}\right)\right)
+    \log(\Gamma_{p}(a)) = C + \displaystyle \sum_{i=1}^{p} \log\left(\Gamma\left(a - \frac{i - 1}{2}\right)\right)
 
-where :math:`C = \log(\pi) \times \frac{p (p - 1)}{2}` and :math:`\gamma(.)` is the Gamma function.
+where :math:`C = \log(\pi) \times \frac{p (p - 1)}{2}` and :math:`\Gamma(.)` is the Gamma function.
```

Comment:
Thanks! Could you change the dot to `\cdot` as well?

Reply:
sure, will use `\cdot` for placeholder